src/com/facebook/buck/cli/TestCommand.java

```diff
@@ -691,4 +707,60 @@ public void printUsage(PrintStream stream) {
   public String getShortDescription() {
     return "builds and runs the tests for the specified target";
   }
+
+  /**
+   * args4j does not support parsing repeated (or delimiter separated) Enums by default.
```

Comment:
is it possible to use `multiValued` attribute instead? I personally would prefer having something like
`--code-coverage-format=HTML --code-coverage-format=XML` to a comma delimited string.

Reply:
I was referring to http://args4j.kohsuke.org/apidocs/org/kohsuke/args4j/Argument.html#multiValued()

Reply:
multiValued attribute support was removed while ago: https://github.com/kohsuke/args4j/issues/24

Let me try updating CoverageReportFormatsHandler to parse multiple values

Reply:
All right, I experimented a little, "--code-coverage-format=HTML --code-coverage-format=XML" is not supported, what you can do though is "--code-coverage-format HTML XML", but I'd argue it is even worse than csv.